lib/burnside/intrinsics.cc

```diff
@@ -21,6 +21,7 @@
 #include "fir/FIRType.h"
 #include "llvm/ADT/Optional.h"
 #include "llvm/ADT/Twine.h"
+#include <algorithm>
```

Comment:
I didn't spot it. What `std` algorithm are you using?

Reply:
`std::lexicographical_compare` in `FunctionDistance::isSmallerThan`. If you do not like this dependency, I can replace it with:
```
for (std::size_t i{0}; i< dataSize; ++i) {
  if (conversions[i] != that.conversions[i] ) {
   return conversions[i] < that.conversions[i];
  }
}
```
Do you want me to get rid of `std::lexicographical_compare` ?

Reply:
Thanks. No leave it. It's fine.